src/components/PostTemplate.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { graphql, useStaticQuery } from "gatsby";
+import PageLayout from "./PageLayout";
+import Post from "./Post";
+import TitleArea from "./TitleArea";
+import DividerLine from "./DividerLine";
+import { colors } from "../_Theme/UpdatedBrandTheme";
+
+const PostTemplate = () => {
+  const { contentfulPost } = useStaticQuery(
+    graphql`
+      query PostQuery($slug: String) {
+        contentfulPost(slug: { eq: $slug }) {
+          slug
+          title
+          content {
+            json
+          }
+        }
+      }
+    `
+  );
+
+  return (
+    <PageLayout title={contentfulPost.title}>
+      <TitleArea title={contentfulPost.title} />
```

Comment:
Looks like the TitleArea isn't working. At least on FF.

<img width="1680" alt="Screen Shot 2020-03-19 at 10 06 25 AM" src="https://user-images.githubusercontent.com/10056079/77094163-54300e80-69c9-11ea-8e37-ace254f0be39.png">